include/triggeralgs/Issues.hpp

```diff
@@ -28,4 +28,10 @@ ERS_DECLARE_ISSUE(triggeralgs,
                   "Bad configuration in " << alg_name,
                   ((std::string)alg_name))
 
+ERS_DECLARE_ISSUE(triggeralgs,
+                  InvalidConfiguration,
```

Comment:
Yeah, I noticed the space earlier on too, I'll correct it.
The main reason why I made a separate `InvalidConfiguration` is because `BadConfiguration` requires algorithm name, which is defined as TRACE_NAME in each algorithm separately -- so it's not available from the base class level.